h/static/styles/mixins/_styled-text.scss

```diff
@@ -111,7 +111,7 @@
   pre code {
     padding: 10px;
     display: block;
-    background-color: $gray-lightest;
+    background-color: $grey-1;
```

Comment:
We don't have $gray-lightest in the palette anymore, so I have replaced it with $grey-1.